tests/dummy/app/router.js

```diff
@@ -1,11 +1,10 @@
 import AddonDocsRouter, { docsRoute } from 'ember-cli-addon-docs/router';
-import RouterScroll from 'ember-router-scroll';
 import config from './config/environment';
 
-const Router = AddonDocsRouter.extend(RouterScroll, {
-  location: config.locationType,
-  rootURL: config.rootURL,
-});
+export default class Router extends AddonDocsRouter {
```

Comment:
`AddonDocsRouter` is actually just `class AddonDocsRouter extends EmberRouterScroll {}` per [`ember-cli-addon-docs/addon/router.js`](https://github.com/ember-learn/ember-cli-addon-docs/blob/v0.10.0/addon/router.js) so it's not needed

Reply:
nice!